workshop/content/flink-on-kda/local-flink-development/anatomy-flink-program.en.md

```diff
@@ -4,15 +4,15 @@ chapter: false
 weight: 40
 ---
 
-According to the [Flink documentation](https://ci.apache.org/projects/flink/flink-docs-stable/dev/api_concepts.html#anatomy-of-a-flink-program), a Flink program consists of the following basic parts:
+According to the [Flink documentation](https://nightlies.apache.org/flink/flink-docs-master/docs/dev/datastream/overview/#anatomy-of-a-flink-program), a Flink program consists of the following basic parts:
     
 1. Obtain an execution environment,
 1. Load/create the initial data,
 1. Specify transformations on this data,
 1. Specify where to put the results of your computations, and
 1. Trigger the program execution
 
-If you are attending an AWS hosted event, we will cover the different parts in a presentation format. If you are running the workshop on your own, please refer to the official [Flink documentation](https://ci.apache.org/projects/flink/flink-docs-stable/dev/api_concepts.html#anatomy-of-a-flink-program) to learn more about the anatomy of a Flink program. 
+If you are attending an AWS hosted event, we will cover the different parts in a presentation format. If you are running the workshop on your own, please refer to the official [Flink documentation](https://nightlies.apache.org/flink/flink-docs-master/docs/dev/datastream/overview/#anatomy-of-a-flink-program) to learn more about the anatomy of a Flink program. 
```

Comment:
Can we please pin this to the last stable verison: https://nightlies.apache.org/flink/flink-docs-stable/docs/dev/datastream/overview/#anatomy-of-a-flink-program